src/api/migrations/0122_mv_invert_attributes.py

```
@@ -0,0 +1,13 @@
+from django.db import migrations
+
+from api.models.view_models.model_view_migrations import forward_sql, reverse_sql
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0121_vw_project_summary_sample_events_macroinvertebrate"),
+    ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)